models/Asset.php

```diff
@@ -258,7 +261,7 @@ public static function getById($id, $force = false)
                 return null;
             }
         } else {
-            \Pimcore\Cache\Runtime::set($cacheKey, $asset);
+            self::setCache($asset, $cacheKey);
```

Comment:
This now saves the non-runtime-cache, too, which wasn't the case before. Is this correct/intended?

Reply:
No, it is not intended. This is still a work in progress.